kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiSessionManager.scala

```diff
@@ -165,6 +173,11 @@ class KyuubiSessionManager private (name: String) extends SessionManager(name) {
     getSessionOption(sessionHandle).map(_.asInstanceOf[KyuubiBatchSessionImpl]).orNull
   }
 
+  private def metadataManager: MetadataManager = {
```

Comment:
we can use metadataManager.foreach for all operations

Reply:
changed